src/examples/deepExamples.cpp

```diff
@@ -147,7 +148,8 @@ writeDeepScanlineFile (
 
     Array2D<half*>& dataA,
 
-    Array2D<unsigned int>& sampleCount)
+    Array2D<unsigned int>& sampleCount,
```

Comment:
Formatting (using openexr's .clang-format)
```suggestion
    Array2D<unsigned int>& sampleCount,
    Compression            compression = Compression::ZIPS_COMPRESSION)
```